stories/dropzone/index.jsx

```diff
@@ -1,30 +1,116 @@
 import DropzoneDOCS from 'components/dropzone/DOCS.md';
 import DropzoneREADME from 'components/dropzone/README.md';
 import React from 'react';
 
 import { decorator } from '../../utils/decorator';
 import { WrappedComponent } from '../helpers/wrapped';
 
+export const DropzoneChild = ({ error, isDragActive, isLoading, content, icon }) => {
+  return (
+    <div style={{ textAlign: 'center', display: 'flex', alignItems: 'center', flexDirection: 'column', gap: '10px' }}>
+      <Icon size={20} icon={icon} />
+      <Text component='p'>{content}</Text>
+      {!error && !isDragActive && !isLoading && (
+        <Text component='p' size='s'>
+          Only mp4 format and max size 100Mb
+        </Text>
+      )}
+    </div>
+  );
+};
+
 export default decorator('Dropzone', DropzoneDOCS, DropzoneREADME).add('Dropzone component', () => {
+  const [showErrMsgInCustomDropzone, setShowErrMsg] = React.useState(false);
+  const dropzoneMapper = {
+    error: {
+      text: 'Error msg',
+      icon: Icon.icons.close,
+    },
+    default: {
+      text: 'Custom dropzone',
+      icon: Icon.icons.upload,
+    },
+    onDrag: {
+      text: 'Drop your file here',
+      icon: Icon.icons.fileMinus,
+    },
+  };
+
   return (
     <WrappedComponent>
-      <Row style={{ justifyContent: 'space-around' }}>
-        <Col xs="2" sm="4" lg="4" style={{ height: '200px' }}>
-          <Dropzone />
+      <Row style={{ justifyContent: 'space-around', margin: '10px auto' }}>
+        <Col xs='2' sm='4' lg='4'>
+          <Dropzone
+            options={{
+              onDrop: (files, rejectedFiles) => {
+                if (rejectedFiles.length > 0) {
+                  setShowErrMsg(true);
+                } else {
+                  setShowErrMsg(false);
+                }
+
+                files.forEach(async (file) => {
+                  const reader = new FileReader();
+
+                  reader.onloadend = function () {
+                    const arrayBufferView = new Uint8Array(reader.result);
+                    // you can send blob if needed to server in a blob format
+                    const blob = new Blob([arrayBufferView], { type: 'video/mp4' });
+                    // or you can use videoUrl for show video material if needed
+                    const videoUrl = URL.createObjectURL(blob);
+                    console.log('CHECK the console', videoUrl);
+                  };
+
+                  reader.onerror = () => {
+                    console.log('error');
+                  };
+
+                  reader.readAsArrayBuffer(file);
+                });
+              },
+              accept: 'video/mp4',
+              maxSize: 100 * 1048576,
+              multiple: false,
+            }}
+            mode='custom'
+            wrapperProps={{ style: { minHeight: '200px' } }}
+          >
+            {(props) => {
+              const info =
+                // eslint-disable-next-line no-nested-ternary
```

Comment:
?????)

Reply:
I won't complain, because it's stories, but I hope you don't ignore it anywhere else

Reply:
what ?) 
it's a shortest way to write condition

Reply:
Yes, but I like many of us don't want to see ternary in project. That's why we have a no-ternary in our eslint
I hope it's only here

Reply:
Writing shortest way !== best solution
Same for naming, etc. 
I hope you know it and understand it🙂

Reply:
if it's used like `if else` - that's ok to have chain like this